src/app-logic/l10n.js

```diff
@@ -5,21 +5,25 @@
 // @flow
 
 import { FluentBundle, FluentResource } from '@fluent/bundle';
-import engFtlFile from '../../locales/en-US/app.ftl';
 
 export const AVAILABLE_LOCALES: Array<string> = ['en-US'];
 export const DEFAULT_LOCALE = 'en-US';
 
-// FIXME: This needs rework, to make it generic. As we'll fetch ftl's depending on users locale.
+/**
+ * Fetches ftl file of different locales.
+ * Returns the locale and the ftl string grouped as an Array.
+ */
 export async function fetchMessages(locale: string): Promise<[string, string]> {
-  const response = await fetch(engFtlFile);
+  const response = await fetch(`/locales/${locale}/home.ftl`);
```

Comment:
This should be `app.ftl` :)

Reply:
Yess. I'm sorry it was so silly. I had to rebase and test the UI it got mixed up. Thanks!